src/client/automation/playback/click/mouseup-command.js

```diff
@@ -0,0 +1,34 @@
+import hammerhead from '../../deps/hammerhead';
+
+const browserUtils   = hammerhead.utils.browser;
+const eventSimulator = hammerhead.eventSandbox.eventSimulator;
+const listeners      = hammerhead.eventSandbox.listeners;
+
+class ElementMouseUpCommand {
```

Comment:
We don't have other commands. So, the packing code to the separate `Command` class is unnecessary.

Reply:
I found it useful, it helps to keep `index.js` clean, since it should not know anything about hacks like `getTimeStamp` or `browserUtils.isIE`

Reply:
I also feel that there is too much code for this. I feel it can be useful to extract timestamp manipulation functions to a separate file, but it's better to use them directly in `_mouseUp` in the click automation.